puppet-client.html

```diff
@@ -162,16 +199,46 @@
                 value: '/'
             },
 
+            /**
+             * @private
```

Comment:
Is it on by default? [docs](https://www.polymer-project.org/1.0/docs/devguide/data-binding#property-notification) doesn't state it one way or the other, but I though it's off